superset/tasks/schedules.py

```diff
@@ -475,3 +638,22 @@ def schedule_hourly() -> None:
     stop_at = start_at + timedelta(seconds=3600)
     schedule_window(ScheduleType.dashboard, start_at, stop_at, resolution)
     schedule_window(ScheduleType.slice, start_at, stop_at, resolution)
+
+
+@celery_app.task(name="alerts.schedule_check")
+def schedule_alerts() -> None:
+    """ Celery beat job meant to be invoked every minute to check alerts """
+
+    # if not config["ENABLE_SCHEDULED_EMAIL_REPORTS"]:
```

Comment:
probably just delete it, I assume this function should not be called if it's not enabled